cpg_workflows/stages/seqr_loader_long_read/long_read_snps_indels_annotation.py

```diff
@@ -0,0 +1,428 @@
+"""
+Workflow for finding long-read SNPs_Indels files, updating file contents, merging, and annotating the results
+"""
+
+from functools import cache
+
+from google.api_core.exceptions import PermissionDenied
+
+from cpg_utils import Path, to_path
+from cpg_utils.config import config_retrieve, image_path
+from cpg_utils.hail_batch import get_batch
+from cpg_workflows.jobs.seqr_loader import annotate_dataset_jobs
+from cpg_workflows.jobs.seqr_loader_snps_indels import (
+    annotate_cohort_jobs_snps_indels,
+    split_merged_vcf_and_get_sitesonly_vcfs_for_vep,
+)
+from cpg_workflows.jobs.vep import add_vep_jobs
+from cpg_workflows.stages.seqr_loader import es_password
+from cpg_workflows.targets import Dataset, MultiCohort, SequencingGroup
+from cpg_workflows.utils import get_logger
+from cpg_workflows.workflow import (
+    DatasetStage,
+    MultiCohortStage,
+    SequencingGroupStage,
+    StageInput,
+    StageOutput,
+    get_workflow,
+    stage,
+)
+from metamist.graphql import gql, query
+
+from ...resources import joint_calling_scatter_count
```

Comment:
```suggestion
from cpg_workflows.resources import joint_calling_scatter_count
```

feels like this should be pulled from the installed module instead of the relative path, which would place it above wit the rest of the cpg_workflows import statements